src/main/java/pl/envelo/moovelo/controller/mapper/AttachmentMapper.java

```diff
@@ -0,0 +1,47 @@
+package pl.envelo.moovelo.controller.mapper;
+
+import pl.envelo.moovelo.controller.dto.AttachmentDto;
+import pl.envelo.moovelo.entity.Attachment;
+import pl.envelo.moovelo.service.CommentService;
+import pl.envelo.moovelo.service.event.EventInfoService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class AttachmentMapper {
+    private final EventInfoService eventInfoService;
+    private final CommentService commentService;
+
+    public AttachmentMapper(EventInfoService eventInfoService, CommentService commentService) {
+        this.eventInfoService = eventInfoService;
+        this.commentService = commentService;
+    }
+
+    public static List<AttachmentDto> mapFromAttachmentListToAttachmentDtoList(List<Attachment> attachments) {
```

Comment:
Odnośnie poprzedniego komentarza, sygnatura metody to : public static AttachmentDto mapAttachmentToAttachmentDto(Attachment attachment) {}